pymc/distributions/timeseries.py

```diff
@@ -616,17 +615,29 @@ def ar_moment(op, rv, rhos, sigma, init_dist, steps, noise_rng):
     return at.full_like(rv, moment(init_dist)[..., -1, None])
 
 
-class GARCH11(distribution.Continuous):
+class GARCH11RV(SymbolicRandomVariable):
+    """A placeholder used to specify a log-likelihood for an AR sub-graph."""
+
+    default_output = 1
+    _print_name = ("GARCH11", "\\operatorname{GARCH11}")
+
+    def update(self, node: Node):
+        """Return the update mapping for the noise RV."""
+        # Since noise is a shared variable it shows up as the last node input
+        return {node.inputs[-1]: node.outputs[0]}
+
+
+class GARCH11(Distribution):
```

Comment:
Should also implement a dispatch moment for GARCH11, otherwise you'll get a warning whenever using this distribution without a manual `initval`

Reply:
You mean a `@_moment.register(GARCH11RV)` right?

Reply:
Yes